src/groovy/org/pih/warehouse/api/DocumentGroupCode.groovy

```diff
@@ -3,7 +3,7 @@ package org.pih.warehouse.api
 enum DocumentGroupCode {
 
     EXPORT('Export'),
-    INVOICE('Invoice'),
+    INVOICE_TEMPLATE('Invoice Template'),
```

Comment:
Can't we have both? I assume someone might upload an INVOICE document.

Reply:
It is `DocumentGroupCode` (not the `DocumentCode`) and I wanted it to correspond to the existing code. However it was before used only in this line https://github.com/openboxes/openboxes/pull/3197/files#diff- 4ee651995dd031b2da56ca8c0c19c734e70e22b7475bf0cda921087d94cfce88L2608 which I removed, and now it is not used anywhere. I guess it can get rid of it too.